### PR TITLE
Fix #2360: Remove unnecessary Initialisation of Firebase app in Testing

### PR DESCRIPTION
**Pull request by etigoel**
## Explanation
Fix #2360 : Remove unnecessary Initialisation of Firebase app in Testing.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by FareesHussain** [comment]
@etigoel please check the checklist in the PR description

**Comment by etigoel** [comment]
> @etigoel please check the checklist in the PR description

Locally the tests are getting passed. Can we check if the tests are flaky.

**Comment by FareesHussain** [comment]
> > @etigoel please check the checklist in the PR description
> 
> Locally the tests are getting passed. Can we check if the tests are flaky.
> 
> 

Add screenshots for the passing tests

File: .idea/misc.xml
Comment (rt4914) [inline .idea/misc.xml]: Remove this line.

Comment (anandwana001) [inline .idea/misc.xml]: there should be no modification in this file.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks to @etigoel  for your first successful pull request. 

1. Please follow the checklist given in the Pull request description
      - PR title
      - PR description

2. Could you please confirm that after removing all the test cases are passing on Espresso? 
For Robolectric the test cases are passing on GitHub action so no need to check locally.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested nit change.
Also, please follow the instructions mentioned in point 4. https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

File: .idea/misc.xml
Comment (etigoel) [reply .idea/misc.xml]: Done

**Comment by etigoel** [comment]
> Thanks to @etigoel for your first successful pull request.
> 
> 1. Please follow the checklist given in the Pull request description
>    
>    * PR title
>    * PR description
> 2. Could you please confirm that after removing all the test cases are passing on Espresso?
>    For Robolectric the test cases are passing on GitHub action so no need to check locally.

To run tests on espresso, I have created Android-instrumented-tests run configuration and its passing. Do we need to perform any other action.

**Comment by etigoel** [comment]
> > > @etigoel please check the checklist in the PR description
> > 
> > 
> > Locally the tests are getting passed. Can we check if the tests are flaky.
> 
> Add screenshots for the passing tests

All the tests are passing currently.

**Comment by etigoel** [comment]
The test is getting passed locally. 
image

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by rt4914** [comment]
Will wait for @anandwana001 approval.

**Comment by rt4914** [comment]
@etigoel Please update title and description as per this comment: https://github.com/oppia/oppia-android/pull/2360#pullrequestreview-561023186

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @etigoel

**Review by rt4914 — CHANGES_REQUESTED** [review]
Blocking because of title and description. 

As per mine and @anandwana001 previous comments please update the title and description of this PR so that you mention the issue number correctly.
Read this wiki and follow the 4th step to fix the PR. https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

**Comment by ShridharGoel** [comment]
@etigoel You also need to update the PR description to include the issue number.

**Comment by etigoel** [comment]
> @etigoel You also need to update the PR description to include the issue number.

For this I am using git commit -m "message" -m "message". Do we need to something else.

**Comment by FareesHussain** [comment]
> > @etigoel You also need to update the PR description to include the issue number.
> 
> For this I am using git commit -m "message" -m "message". Do we need to something else.

@etigoel You need to do this in github go to this PR description and edit
image

**Comment by Luffy18346** [comment]
Hi @etigoel, as @FareesHussain suggested above, you have to edit the PR and update the explanation.
For example, write "Fixes #(bug number): " (or "Fixes part of #(bug number)" if the PR only partially fixes an issue) below the explanation when you edit the PR. If you want, you can add further description about what issue this PR solves.

For the reference, you can take a look at 4th point in https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

**Comment by etigoel** [comment]
> Hi @etigoel, as @FareesHussain suggested above, you have to edit the PR and update the explanation.
> For example, write "Fixes #(bug number): " (or "Fixes part of #(bug number)" if the PR only partially fixes an issue) below the explanation when you edit the PR. If you want, you can add further description about what issue this PR solves.
> 
> For the reference, you can take a look at 4th point in https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

Done.. Thank you

**Comment by etigoel** [comment]
> > > @etigoel You also need to update the PR description to include the issue number.
> > 
> > 
> > For this I am using git commit -m "message" -m "message". Do we need to something else.
> 
> @etigoel You need to do this in github go to this PR description and edit
> image

Done.. Thanks

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**
